connectors/src/connectors/notion/lib/notion_api.ts

```diff
@@ -729,6 +729,14 @@ function parsePropertyText(
         ? property.title.map((t) => t.plain_text).join(" ")
         : null;
     case "rich_text":
+      if (!property.rich_text.map) {
```

Comment:
can we do if `!Array.isArray(property.rich_text)` ? Your test will still crash if `rich_text` is `undefined` ?

Reply:
Also, if it's indeed not an array, let's return `null` ?

Reply:
code updated